react-frontend/src/components/Guides/CollaborationTools/collaborationTool.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function CollaborationTool(props) {
+  return (
+    <div className="collaborationTool">
+      <a href={props.url}>
+        <strong>{props.name}</strong>
+      </a>{' '}
+      *
```

Comment:
whats the * for?

Reply:
It was in the original formatting of the digital site, I believe they wanted an asterix there